mbuild/formats/lammpsdata.py

```diff
@@ -642,18 +774,36 @@ def write_lammpsdata(
 
                 else:
                     data.write("\nAngle Coeffs # harmonic\n")
-                    data.write("#\treduced_k\t\ttheteq(deg)\n")
+
+                    if unit_style == "lj":
+                        data.write("#\treduced_k\t\ttheteq(deg)\n")
+                    else:
+                        data.write("#\tk(kcal/mol/rad^2)\t\ttheteq(deg)\n")
+
                     for params, idx in sorted_angle_types.items():
-                        data.write(
-                            "{}\t{}\t\t{:.5f}\t# {}\t{}\t{}\n".format(
-                                idx,
-                                params[0],
-                                params[1],
-                                params[3][0],
-                                params[2],
-                                params[3][1],
+                        # If the user specified LJ unit style, revert the internal conversion of k by ParmEd
+                        if unit_style == "lj":
+                            data.write(
```

Comment:
Same comment as above with `_get_angle_types` function?

Reply:
See my comment above about the bond conversions